package.json

```
@@ -30,7 +30,7 @@
     "zod": "^3.21.4"
   },
   "devDependencies": {
-    "@openmrs/esm-framework": "next",
+    "@openmrs/esm-framework": "^5.6.1-pre.1895",
```

```suggestion
    "@openmrs/esm-framework": "next",
```
we only need to update the `yarn.lock` file for such updates

---

I have issue with navigate i decided to use` yarn up openmrs @openmrs/esm-framework` and it worked